### PR TITLE
Feature/test build

### PR DESCRIPTION
**Pull request by gauthiermartin**
## Description
Create a base image to ease up build time using CI/CD

## Fixes
Link all the related issues from the issue tracker
- Closes: #1 

## How has this been tested ?
Local and online build

## Checklist
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,674 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```
Comment (marcantoinecouture) [inline LICENSE:1]: Is it a mistake that licence file is deleted?

**Review by marcantoinecouture — APPROVED** [review]
👍 One comment on Licence file

File: LICENSE (hunk 1 above)
Comment (gauthiermartin) [reply LICENSE:1]: No we will not produce an entrypoint for this repository since multiple licence file are embeded in the container by NVIDIA

**Review by marcantoinecouture — APPROVED** [review]
👍

**Pull request merged**